tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -112,10 +111,9 @@ export const downloadResources = async (args: downloadResourcesArgs): Promise<Do
 
   switch (via) {
     case 'SIDEBAR_PANEL': {
-      if (folder) {
+      if (folder && folder !== 'undefined') {
```

Comment:
are you sure that folder will have string `undefined`?

Reply:
yes if there's no `from` column in data table
https://github.com/owncloud/web/blob/841d6af28b2b34f23138ce8c6b0244098ce4a978/tests/e2e/cucumber/features/integrations/link.feature#L38-L41

Reply:
@SwikritiT see https://github.com/owncloud/web/pull/6889#discussion_r871032039